wrapanapi/systems/msazure.py

```diff
@@ -742,6 +743,23 @@ def remove_pips_by_search(self, pip_template, resource_group=None):
             self.logger.debug('No PIPs matching "%s" template were found', pip_template)
             return results
 
+    def remove_discs_by_search(self, disc_name):
+        """
+        Used for clean_up jobs to remove discs that are left after deleting the VM
+        """
+        self.logger.info('Attempting to find the disc image {}'.format(disc_name))
+        results = []
+        path = 'Microsoft.Compute/Images/templates/'
```

Comment:
Is this necessarily the path? I guess I don't know your automation setup, but thought I would ask.

Reply:
Also, did you check out `list_compute_images` for listing things in 'Microsoft.Compute/Images'? I would consider enhancing it, or wrapping find_compute_images() around it, to account for the performance difference between searching explicitly and just listing and then finding in the list.

Reply:
@djberg96 Thanks Dan, all the discs that were deleted had this path and it seems to be the path for such kind of discs. Other images have different path.

Reply:
@mshriver `list_compute_images` shows images that were created as images. not these leftovers from vm deletion.

I specified the container as all these discs end up in the `system` container